Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM opensuse/amd64:42.3
+FROM registry.suse.de/non_public/infrastructure/images/opensuse_leap_15.0/images/opensuse-leap-15.0:current
```

Comment:
Should we be using non-public images in a public git repo?

Reply:
you are right, we will try to find alternative (public Leap 15 image)

Reply:
The official image is at https://build.opensuse.org/package/binaries/openSUSE:Leap:15.1:Images/opensuse-leap-image:docker/images, not sure if docker can consume such an URL without registry, though.

Reply:
thanks. we will rollback the latest 2 commits since we want to simplify the Docker setup in a separate PR, but there we will try to use that image